intermediate result

gkanonimus opened the pull request:
Скидываю то, что есть на данный момент. Это промежуточный результат, я работаю, буду отправлять по мере готовности и работоспособности. Прошу прощения за столь поздний инвайт в репозиторий

gkanonimus committed:
intermediate result

Скидываю то, что есть на данный момент. Это промежуточный результат, я работаю, буду отправлять по мере готовности и работоспособности. Прошу прощения за столь поздний инвайт в репозиторий

gkanonimus commented:
Проект сыпет очень много ошибок, пытаюсь в ускоренном темпе их исправлять и дорабатывать свою ORM

MarinaPaley reviewed (approved):
Сразу много. Проверять в github такой проект невозможно.
В качестве черновика - хорошо.
По классам. Дорабатывайте необходимые методы, проверки и т.д. Пишите на все тесты.
По маппингу. Я еще и не рассказывала))))

gkanonimus committed:
final result

Понял, что не могу написать на шарпе, VS Code отказывается устанавливать связи между файлами, пришлось переобуться и написать на Python

gkanonimus commented:
Добрый вечер, Марина Алексеевна! Я понял, что у меня не получится написать на C#, и решил попробовать на Python - всё получилось, тесты проходятся всё, прикрепляю с файлами отчёт о проделанной работе

MarinaPaley reviewed (changes requested):
Титул должен быть по ГОСТ по НИР 2017 (Москва 2023) . без тире, г. и т.д.
Название рисунков - с большой буквы
Вставлять картинкой скрипт... ну, такое((((
Рисунки вставлены неправильно. [Здесь пример работы с Word](https://gist.github.com/MarinaPaley/1f83a9a15ae0a36f740eefe573a86f46)
"Изучены и закреплены основные аспекты построения защищенных баз данных на примере PostgreSQL." Расскажите что это ?))))

gkanonimus committed:
Обновление отчета

gkanonimus commented:
Скрипт у меня не в картинках, это оформление для удобного чтения кода. Остальные правки учёл, обновил файл отчета

MarinaPaley reviewed (approved):
Вывод ужасный. Я бы так не писала. кто сказал, что ORM производительный? ORM для.. магазина &ndash; совсем звучит плохо.
оформление кода на черном фоне &ndash; плохо, так как явно говорит, что Вы не можете поменять тему для среды разработки (для отчета)

gkanonimus committed:
Report updated

gkanonimus commented:
Исправил отчёт, поменяв фон для кода и изменив вывод по работе
